### ISSUE TITLE
Can I ask some questions about TOOD/mmdet/core/bbox/assigners/task_aligned_assigner.py ?

### ISSUE THREAD

**Issue by boy-cyh** [fcjian/TOOD#6]
line 65: decode_bboxes (Tensor): predicted bounding boxes, shape(n, 80)
Why  the shape of bbox  here is (n,80)?
I started reading from tood_head.py and I wonder why the shape here is not the (n,4)?

**Comment by fcjian** [comment]
You are right. It is (n,4). I made a typing error. Thank you for your reminder.

**Comment by boy-cyh** [comment]
Thank you for your answer！
And this is associated with a line of code.
the 83 line in the task_aligned_assigner.py : alignment_metrics = bbox_scores ** alpha * overlaps ** beta
In my inference，the shape of  “bbox_scores” is (n,1,80)  and the shape of "overlaps" is (n ,num_gts).
But the "num_gts"  is not equal to 80，It seems that the two cannot be multiplied by "*"。
It's a little confusing to me.

**Comment by boy-cyh** [comment]
Sorry, I think of gt_label as None.I understand it.
